README.md

```diff
@@ -10,150 +10,148 @@
 
 A GitHub plugin for Mattermost. Supports GitHub SaaS and Enterprise versions.
 
+## Table of Contents
+
+ - [1. Audience](#1-audience)
```

Comment:
These don't seem to work when testing on https://github.com/mattermost/mattermost-plugin-github/blob/justinegeffen-github-revision/README.md. Do they work on you local instance?